test/cypress/features/delegates/delegates.js

```diff
@@ -0,0 +1,138 @@
+import { ss } from '../../../constants';
+
+const getDelegateNameFromRow = (ele) => ele.find('span:first-child > div > div:last-child > div:last-child > p:first-child').text();
+
+/* eslint-disable */
+Then(/(\w+) count should have value greater than (\d+)/, function(displayElementClassName, displayContent){
```

Comment:
@eniolam1000752 Please move `eslint-disable` to the top of the file and import `Then` and `When` statements from cucumber

Reply:
even with that, there is this lint error A function with a name starting with an uppercase letter should only be used as a constructor. which I think can only be resolved if Then and When are declared as constructors